### DISCUSSION TITLE
After redirect Toast message

### DISCUSSION THREAD

**Discussion by JBiezais** [laravel/nova-issues#5434] [Ideas & Feature Requests] [answered]
On Resource delete action, handle method returns to its parent 
```
public function handle(ActionFields $fields, Collection $models): array
{
    ...
    return Action::visit('/resources/'.Resource::uriKey().'/'.$parent->id);
}
```
How can I pass a toast message like "This record has been deleted" with it?

**Comment by arturasfrontit** [comment]
Any updates?

**Comment by ghost** [comment]
Ability to redirect AND show a message would be an appreciated addition to the Nova actions.
Another post that relates to this topic - https://github.com/laravel/nova-issues/issues/1961

**Reply by arturasfrontit** [reply]
I want that to use it in middleware

**Comment by RealityGamin** [comment]
Would love to hear some ideas or updates on this, even a hacky solution would be appreciated !

**Comment by crynobone** [comment]
Do note that not all actions/requests in Laravel Nova trigger a "redirect". Most would only fetch contents without going through Inertia and it might be best to have it handled using Laravel broadcast with `laravel-echo`

**Reply by ghost** [reply]
I was also considering to opt for Laravel Echo as an alternative. But it would be nice to have something like `Action::visit('path')->withMessage('message')` without the hassle of setting up broadcasting and custom Nova tool.

**Reply by RealityGamin** [reply]
Absolutely this would be beautiful to have!

**Comment by ghost** [comment]
Hi @crynobone. Any chance this one made its way into the roadmap?

**Comment by ghost** [comment, marked as answer]
Just noticed that this feature has been implemented in [v4.27.0](https://nova.laravel.com/releases/4.27.0).

> Nova now allows combining the message and danger methods with the modal, download, deleted, and visit methods.

An example:
```php
return Action::visit('resources/users')->withDangerMessage('Oh, my...');
```

Thanks!
